### DISCUSSION TITLE
Currency Data Source and Accuracy #1

### DISCUSSION THREAD

**Discussion by adesokan01** [adesokan01/T24S2T1EXTG01#5] [General]
What API should we use for real-time currency conversion rates, and what measures should we take to ensure data accuracy and reliability?  Should we consider using a paid API to guarantee the accuracy and up-to-date currency rates, or is a free API sufficient for our project's scope?

**Comment by acantley** [comment]
A good consideration moving forward. There are free APIs available but I would need to do further research on their viability versus paid APIs. For now, I think it we should focus on getting the logic of the calculator sound based on the Conversion Rate table provided, and then consider real-time conversion functionality.
